.github/workflows/main.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           persist-credentials: false
 
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
```

Comment:
This repo is behind on some of the versions of Actions that are updated in the other repos. I figured I would update them while changing this file to match what we have everywhere else. The run still passes without issues on the upgrade